draft-ietf-quic-multipath.md

```diff
@@ -1266,9 +1268,9 @@ Maximum Path Identifier:
   This value MUST NOT exceed 2^32-1, as Path IDs are defined with a maximum value 2^32-1
   as the 32 bits of the Path ID are used to calculate the nonce (see {{multipath-aead}}).
   The Maximum Path Identifier value MUST NOT be lower than the value
-  advertised in the initial_max_path_id transport parameter. Receipt
-  of an invalid Maximum Path Identifier value MUST be treated as a
-  connection error of type MP_PROTOCOL_VIOLATION.
+  advertised in the initial_max_path_id transport parameter.
```

Comment:
```suggestion
  advertised in the initial_max_path_id transport parameter.
  
```

Reply:
Do we need to restate the AEAD reason here? I would rather only have that in the ".Introduction of an Explicit Path Identifier." The text should be:

This value MUST NOT exceed 2^32-1, the maximum value of the Path ID (see {{introduction-of-an-explicit-path-identifier}}).
The Maximum Path Identifier value MUST NOT be lower than the value
advertised in the initial_max_path_id transport parameter.